components/prism-graphql.js

```diff
@@ -27,17 +37,44 @@ Prism.languages.graphql = {
 		pattern: /[a-z_]\w*(?=\s*(?:\((?:[^()"]|"(?:\\.|[^\\"\r\n])*")*\))?:)/i,
 		greedy: true
 	},
+	'constant': /\b(?!ID\b)[A-Z][A-Z_\d]*\b/,
 	'class-name': {
-		pattern: /(\b(?:enum|implements|interface|on|scalar|type|union)\s+|&\s*)[a-zA-Z_]\w*/,
-		lookbehind: true
+		pattern: /(\b(?:enum|implements|interface|on|scalar|type|union)\s+|&\s*|:\s*|\s*\[)[A-Z_]\w*/,
+		lookbehind: true,
+		greedy: true
 	},
 	'fragment': {
 		pattern: /(\bfragment\s+|\.{3}\s*(?!on\b))[a-zA-Z_]\w*/,
 		lookbehind: true,
 		alias: 'function'
 	},
-	'keyword': /\b(?:directive|enum|extend|fragment|implements|input|interface|mutation|on|query|repeatable|scalar|schema|subscription|type|union)\b/,
+	'property-mutation': {
+		pattern: /\b(mutation.*(?:[\n\r\u2028\u2029][\n\s]*)?)\b\w+/g,
```

Comment:
1. I commend you for acknowledging that `.` is equivalent to `[^\n\r\u2028\u2029]` but we can make this pattern a little simple using a line assertion `$`. This works because `/$/m` is equivalent to `/(?!.)/`.
2. `[\n\s]`: `\n` is included by `\s`.
3. Prism will automatically add the `g` flag if necessary. No need to explicitly add it.

```suggestion
		pattern: /\b(mutation.*$\s+)\b\w+/m,
```

Reply:
I can't take credit for knowing the equivalence... the patterns tests pointed this out to me as a fix for what I had before... 😂 fixed now, thanks!